WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailFragment.java

```diff
@@ -39,22 +48,33 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class LoginEmailFragment extends LoginBaseFormFragment<LoginListener>
-        implements TextWatcher, OnEditorCommitListener {
+import static android.app.Activity.RESULT_OK;
+
+public class LoginEmailFragment extends LoginBaseFormFragment<LoginListener> implements TextWatcher,
+        OnEditorCommitListener, GoogleApiClient.ConnectionCallbacks, GoogleApiClient.OnConnectionFailedListener {
     private static final String KEY_GOOGLE_EMAIL = "KEY_GOOGLE_EMAIL";
+    private static final String KEY_HAS_DISMISSED_EMAIL_HINTS = "KEY_HAS_DISMISSED_EMAIL_HINTS";
+    private static final String KEY_IS_DISPLAYING_EMAIL_HINTS = "KEY_IS_DISPLAYING_EMAIL_HINTS";
     private static final String KEY_IS_SOCIAL = "KEY_IS_SOCIAL";
     private static final String KEY_OLD_SITES_IDS = "KEY_OLD_SITES_IDS";
     private static final String KEY_REQUESTED_EMAIL = "KEY_REQUESTED_EMAIL";
+    private static final String LOG_TAG = LoginEmailFragment.class.getSimpleName();
+    private static final int GOOGLE_API_CLIENT_ID = 1001;
+    private static final int REQUEST_CREDENTIALS = 9001;
```

Comment:
Request codes (`9001` in this case) are not really globally uniquely defined so, how about moving this definition to `RequestCodes.java`?

Reply:
I moved the request code in f1728912cda069b3c1df67443f69e7ba4666acee.